Makefile

```diff
@@ -152,9 +158,94 @@ unit-test: go-install
 coverage: unit-test
 	./build/scripts/coverage.sh html
 
+#
+# Tests-related tasks
+#
+CLUSTER_NAME = verrazzano-operator
+CERTS = build/verrazzano-operator-cert
+VERRAZZANO_NS = verrazzano-system
+DEPLOY = build/deploy
+K8RESOURCE = test/k8resource
+
 .PHONY: integ-test
-integ-test: go-install
-	$(GO) test -v ./test/integ/ -timeout 30m --kubeconfig=${KUBECONFIG} --namespace=${K8S_NAMESPACE} --runid=${INTEG_RUN_ID}
+integ-test: create-cluster
+
+	echo 'Create CRDs needed by the verrazzano-operator...'
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanomanagedclusters_crd.yaml
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanomodels_crd.yaml
+	kubectl create -f vendor/${CRDGEN_PATH}/${CRD_PATH}/verrazzano.io_verrazzanobindings_crd.yaml
+	kubectl create -f vendor/${COH_PATH}/${CRD_PATH}/verrazzano.io_cohclusters_crd.yaml
+	kubectl create -f vendor/${HELIDON_PATH}/${CRD_PATH}/verrazzano.io_helidonapps_crd.yaml
+	kubectl create -f vendor/${WKO_PATH}/${CRD_PATH}/verrazzano.io_wlsoperators_crd.yaml
+	kubectl create -f vendor/${VMI_PATH}/${VMI_CRD_PATH}/verrazzano-monitoring-operator-crds.yaml
+
+	echo 'Deploy local cluster and required secret ...'
+	# Create the local cluster secret with empty kubeconfig data.  This will force the verrazzano-operator
+	# to use the in-cluster kubeconfig to access the managed cluster.
+	kubectl apply -f ./test/k8resource/local-cluster-secret.yaml
```

Comment:
is it possible to create a secret with kubectl create secret, since it's an empty secret?